src/token/token.service.ts

```diff
@@ -149,8 +169,19 @@ export class TokenService {
       });
   }
 
+  async getStringifiedToken(token: string, clientIp: string): Promise<any> {
+    if (
+      this.configService.get<string>('app.env') === 'development' &&
+      this.configService.get<string>('bypass.bypassed')
+    ) {
```

Comment:
FOR THE ABOVE LOGIC...rule of thumb if the same block of code is used more than 2 times refactor into a function and then call the function so that the same logic is encapsulated and only ever has to change in 1 place if needed in the future